jwst/assign_wcs/util.py

```diff
@@ -117,7 +117,7 @@ def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=Non
         transform = []
         wcsinfo = pointing.wcsinfo_from_model(refmodel)
         sky_axes, spec, other = gwutils.get_axes(wcsinfo)
-        rotation = astmodels.AffineTransformation2D(np.array(wcsinfo['PC']))
+        rotation = astmodels.AffineTransformation2D(wcsinfo['PC'])
```

Comment:
Is this the line that actually fixes the swapping of the axis order? I assume the previous application of np.array() was doing the swapping?

Reply:
No, that line is just an unnecessary casting of the wcsinfo into a numpy array.  But the API of `pointing.wcsinfo_from_model` already returns this as a numpy array, so no need to have it in there twice.

The actual repair is the deletion of line 56 in resample_utils.py that did an unnecessary swapping of the order.  The order had already been swapped.  I clarified this behavior in the docstrings on the called functions.